source/PlayerInfoPanel.cpp

```diff
@@ -44,6 +46,8 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 
 using namespace std;
 
+class Color;
+
```

Comment:
Why this addition? There are some `Color &` items in this file already.
```suggestion
```

Reply:
Not sure. I might have had this in a different file at first where it was needed, then decided to move it here.